docs/architecture.md

```diff
@@ -69,108 +69,128 @@ The authorization policies evaluated in phase (iii) can use any info from the au
   <thead>
     <tr>
       <th colspan="2">Feature</th>
+      <th>Description</th>
```

Comment:
Most of the changes in this file do not belong to this PR, except because I wanted to add the description about the Festival Wristbands.

I plan to have another iteration soon where I'll extract this table to either its own doc page or back to the main README for immediate access of the list of features.